### PR TITLE
Db/visual test gh action

### PR DESCRIPTION
**Pull request by goransle**
Moved Cypress visual tests to GH actions

### CODE REVIEW

**Review by sebastianbochan — APPROVED** [review]

**Review by pawellysy — APPROVED** [review]

File: .github/workflows/dashboards-visual-test.yml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+
+      - name: Install dependencies
+        uses: cypress-io/github-action@v5
+        with:
+          runTests: false
+
+      - name: Build highcharts
```
Comment (bre1470) [inline .github/workflows/dashboards-visual-test.yml:39]: `Build dashboards` job is missing

File: .github/workflows/dashboards-visual-test.yml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        with: 
+          name: reference
+          path: cypress # this path is root relative, not based on the working-directory above
+
+      - name: Install dependencies
+        uses: cypress-io/github-action@v5
+        with:
+          runTests: false
+
+      - name: Build highcharts
```
Comment (bre1470) [inline .github/workflows/dashboards-visual-test.yml:90]: Same here.

**Review by bre1470 — CHANGES_REQUESTED** [review]
PR needs an update

**Review by pawellysy — APPROVED** [review]
lgtm

**Review by bre1470 — APPROVED** [review]
Thank you! Looks good. 👍

**Review by oysteinmoseng — APPROVED** [review]

**Pull request merged**